osu.Game/Rulesets/Edit/PlacementBlueprint.cs

```diff
@@ -171,7 +171,7 @@ public virtual void UpdateTimeAndPosition(SnapResult result)
         /// </summary>
         protected void ApplyDefaultsToHitObject() => HitObject.ApplyDefaults(beatmap.ControlPointInfo, beatmap.Difficulty);
 
-        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => Parent?.ReceivePositionalInputAt(screenSpacePos) == true;
+        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => true;
```

Comment:
There's a catch-local override that does the same:

https://github.com/ppy/osu/blob/99d4d2be375a9a2d7d2f99cec133eadc0fc3a43e/osu.Game.Rulesets.Catch/Edit/Blueprints/CatchPlacementBlueprint.cs#L28

Could probably get rid of it while we're here.